dat/BIOS (Non-Merged).dat

```diff
@@ -113,24 +105,21 @@ game (
 	rom ( name st011.data.rom size 4096 crc 803256a7 md5 10bd3f4aa949737ab9836512c35bcc29 sha1 798ab84110ca93a6e73a540f4146d21f8d9e1442 )
 	rom ( name st011.program.rom size 49152 crc a741e9fb md5 95222ebf1c0c2990bcf25db43743f032 sha1 375eca92592621e83881f3a9d7fd5ea3e5b306dc )
 	rom ( name st018.data.rom size 32768 crc b5255459 md5 49c898b60d0f15e90d0ba780dd12f366 sha1 b19c0f8f207d62fdabf4bf71442826063bccc626 )
-	rom ( name st018.program.rom size 131072 crc f73d5e10 md5 dda40ccd57390c96e49d30a041f9a9e7 sha1 388e3721b94cd074d6ba0eca8616523d2118a6c3 )
-	rom ( name BS-X.bin size 1048576 crc f51f07a0 md5 fed4d8242cfbed61343d53d48432aced sha1 604556b2e62860af18db5a77f2956ebc75450020 )
```

Comment:
BS-X.bin is used in SNES9X....
https://github.com/libretro/snes9xherd/blob/0d34187679216d3cb96eed9c862958c7f6eeb14f/src/bsx.c#L966